lichbot.lic

```diff
@@ -55,6 +55,7 @@ class Lichbot
     @last_action = Time.now
     @last_teacher = nil
     @inventory = []
+    @bot_bank = UserVars.bot_bank || {}
```

Comment:
Fair. Also, I agree with the point made by E... this isn't a good way to handle folk's coins reliably. It should be using a file at the very least.